src/pages/getinvolved.jsx

```diff
@@ -209,66 +213,258 @@ export const GetInvolved = () => {
         <TechStackList />
       </Box>
 
-      <div className={getinvolvedStyles.wrapper}>
-        <Image
-          className={getinvolvedStyles.hackathon}
-          src={roadmap}
-          alt="Current Roadmap"
+      <Container maxWidth="md">
+        <Box px={{ xs: '50px', md: '20px' }}>
+          <Typography
+            variant="p"
+            fontSize="30px"
+            align="justify"
+            paragraph={true}
+            my="40px"
+            textAlign="center"
+            lineHeight="30px"
+          >
+            Current Roadmap
+          </Typography>
+          <TransformWrapper initialScale={1}>
+            {({ zoomIn, zoomOut, resetTransform, ...rest }) => (
+              <>
+                <Box
+                  sx={{
+                    my: '20px',
+                    textAlign: 'center',
+                  }}
+                >
+                  <Button
+                    variant="outlined"
+                    startIcon={<AddIcon />}
+                    sx={{ m: 1 }}
+                    onClick={() => zoomIn()}
+                  >
+                    Zoom in
+                  </Button>
+                  <Button
+                    variant="outlined"
+                    startIcon={<RemoveIcon />}
+                    sx={{ m: 1 }}
+                    onClick={() => zoomOut()}
+                  >
+                    {' '}
+                    Zoom out
+                  </Button>
+                  <Button
+                    variant="outlined"
+                    sx={{ m: 1 }}
+                    startIcon={<RestartAltIcon />}
+                    onClick={() => resetTransform()}
+                  >
+                    Reset
+                  </Button>
+                </Box>
+                <TransformComponent>
+                  <Image src={roadmap} alt="roadmap" />
+                </TransformComponent>
+              </>
+            )}
+          </TransformWrapper>
+          <Typography
+            variant="p"
+            fontSize="16px"
```

Comment:
seeing fontSize of 16px and 20px which are fairly similar. Not sure if intention